### PR TITLE
Add support for STIX EDH ISA markings

### PR DESCRIPTION
**Pull request by sddj**
- import the stix_edh package for ISA markings
- use string_types when checking if objects are strings (note parenthesis error at original line 933, new line 939)
- bump stix to version 1.1.1.8 for compatibility with the stix-edh package

### CODE REVIEW

**Comment by jtschichold** [comment]
Hi @sddj,
thanks for this! I will merge this into the next `post` release.

**Comment by jtschichold** [comment]
BTW, could you add your name to the `AUTHORS` file?

File: minemeld/ft/taxii.py [hunk 1]
```diff
@@ -57,6 +60,9 @@
 from . import actorbase
 from .utils import dt_to_millisec, interval_in_sec, utc_millisec
 
+
+del stix_edh  # stix_edh is imported to register the EDH data marking extensions, but it is not directly used
```
Comment (jtschichold) [inline minemeld/ft/taxii.py]: Any particular reason we want to delete it even if not being used directly?

Comment (sddj) [reply minemeld/ft/taxii.py]: This silences a warning about the import being unnecessary and prevents the 'Optimize Imports' in PyCharm from removing the import.

Comment (jtschichold) [reply minemeld/ft/taxii.py]: please could you add this note to the comment on the line? for posterity

**Comment by jtschichold** [comment]
:shipit:

**Pull request merged**